docs/reference/esql/esql-across-clusters.asciidoc

```diff
@@ -19,6 +20,70 @@ include::{es-ref-dir}/search/search-your-data/search-across-clusters.asciidoc[ta
 
 include::{es-ref-dir}/search/search-your-data/search-across-clusters.asciidoc[tag=ccs-proxy-mode]
 
+[discrete]
+[[esql-ccs-security-model]]
+==== Security model
+
+{es} supports two security models for cross-cluster search (CCS):
+
+* <<esql-ccs-security-model-certificate, TLS certificate authentication>>
+* <<esql-ccs-security-model-api-key, API key authentication>>
+
+[discrete]
+[[esql-ccs-security-model-certificate]]
+===== TLS certificate authentication
+
+TLS certificate authentication makes sense in single administrator scenarios where you have full control over both clusters.
```

Comment:
Do we talk about licenses explicitly in docs given variance between self-managed and cloud?

Reply:
I don't know our general stance, but it seems like an important detail since ES|QL is not gated by an enterprise license. IIUC in cloud user have access to all features, but are billed based on usage.

Reply:
We use generic messaging about licensing as a rule, and the main pages for these security models have prerequisites sections that say "_The local and remote clusters must have an appropriate license. For more information, refer to https://www.elastic.co/subscriptions_". So I will refrain from mentioning licensing in this doc.